### PR TITLE
Libvirt daemon

### PR DESCRIPTION
**Pull request by praveenkumar**

### CODE REVIEW

File: provision.sh [hunk 1]
```diff
@@ -7,13 +7,7 @@ set -x
...
 sudo mv /tmp/tools/* /usr/local/bin
 
-sudo dnf install -y libvirt libvirt-devel libvirt-client git libvirt-daemon-kvm bind-utils jq gcc-c++
-
-# Install golang
-curl -L https://dl.google.com/go/go1.13.8.linux-amd64.tar.gz -o go1.13.8.linux-amd64.tar.gz
-tar -xvf go1.13.8.linux-amd64.tar.gz
-sudo mv go /usr/local
-export PATH=$PATH:/usr/local/go/bin
+sudo dnf install -y libvirt libvirt-devel libvirt-client git libvirt-daemon-kvm bind-utils jq gcc-c++ golang
```
Comment (cfergeau) [inline provision.sh:10]: do you still need `gcc-c++`?

File: provision.sh [hunk 2]
```diff
@@ -25,14 +25,12 @@ sudo sysctl -p /etc/sysctl.d/99-ipforward.conf
...
 auth_tcp="none"
-tcp_port = "16509"
-EOF
-sudo bash -c 'cat >> /etc/sysconfig/libvirtd' << EOF
-LIBVIRTD_ARGS="--listen"
 EOF
+
+sudo systemctl enable libvirtd-tcp.socket
+sudo systemctl start libvirtd-tcp.socket
+
```
Comment (cfergeau) [inline provision.sh:33]: no restart of `libvirtd.service` if it's already running? Or are we sure it's not going to be running when `provision.sh` runs?

**Review by cfergeau — APPROVED** [review]
Looks good to me.

File: tools/create-cluster [hunk 3]
```diff
@@ -17,14 +17,6 @@ if [[ ! -z "$OPENSHIFT_INSTALL_RELEASE_IMAGE_OVERRIDE" ]]; then
...
 fi
 
-# https://github.com/ironcladlou/openshift4-libvirt-gcp/issues/29
-# gcp image is provisioned with this version, but is auto-updated.
-# rather than turn off auto-upates, downgrade qemu-kvm each go.
-if ! sudo dnf info qemu-kvm | grep -A 5 'Installed Packages' | grep 88.module+el8.1.0+5708+85d8e057.3; then
-    echo "downgrading qemu-kvm to version 2.12.0-88.module+el8.1.0+5708+85d8e057.3"
-    sudo dnf remove -y qemu-kvm && sudo dnf install -y qemu-kvm-2.12.0-88.module+el8.1.0+5708+85d8e057.3
-fi
-
```
Comment (sallyom) [inline tools/create-cluster:27]: you've verified this is no longer required?

**Review by sallyom — CHANGES_REQUESTED** [review]
much of this is duplicating the already open https://github.com/ironcladlou/openshift4-libvirt-gcp/pull/41  and want to make sure you verified we no longer need to downgrade qemu-kvm version

File: tools/create-cluster (hunk 3 above)
Comment (cfergeau) [reply tools/create-cluster:27]: The linked bugzilla indicate that it's fixed with `qemu-kvm-4.2.0-19.module+el8.2.0+6296+6b821950` and newer, and that 8.3 has something newer than this, so it should no longer be required.

Comment (praveenkumar) [reply tools/create-cluster:27]: @sallyom Yes I did verified this one, you can check same for `openshift-gce-devel` project with updated image.

**Comment by praveenkumar** [comment]
> much of this is duplicating the already open #41 and want to make sure you verified we no longer need to downgrade qemu-kvm version

@sallyom yes, the libvirt daemon changes are duplicate.

File: tools/create-cluster (hunk 3 above)
Comment (sallyom) [reply tools/create-cluster:27]: nice, ok great

**Comment by sallyom** [comment]
@praveenkumar @cfergeau feel free to merge this, I'll rebase #41 to bring in the fix to the teardown script

/lgtm

**Pull request merged**